src/pages/PullRequestPage/Header/HeaderTeam/hooks/usePullHeadDataTeam.tsx

```diff
@@ -0,0 +1,168 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api'
+import A from 'ui/A'
+
+const CoverageObjSchema = z.object({
+  percentCovered: z.number().nullable(),
+})
+
+const ComparisonSchema = z.object({
+  __typename: z.literal('Comparison'),
+  patchTotals: CoverageObjSchema.nullable(),
+})
+
+const CompareWithBaseSchema = z.discriminatedUnion('__typename', [
+  ComparisonSchema,
+])
+
+const RepositorySchema = z.object({
+  __typename: z.literal('Repository'),
+  pull: z
+    .object({
+      pullId: z.number().nullable(),
+      title: z.string().nullable(),
+      state: z
+        .union([z.literal('OPEN'), z.literal('CLOSED'), z.literal('MERGED')])
+        .nullable(),
+      author: z
+        .object({
+          username: z.string().nullable(),
+        })
+        .nullable(),
+      head: z
+        .object({
+          branchName: z.string().nullable(),
+          ciPassed: z.boolean().nullable(),
+        })
+        .nullable(),
+      updatestamp: z.string().nullable(),
+      compareWithBase: CompareWithBaseSchema.nullable(),
```

Comment:
Hook is "identical" to `usePullHeadData` aside from the this field (and renaming the query and all that)